integration_tests/test_rigidbody.py

```diff
@@ -103,6 +102,7 @@ def test_rigidbody_mpi(rigidbody_module):
     sampling = 5
```

Comment:
In default parameters, `ntrials = 10`.
`ntrials`: $min 1 / $max 100 / $title Number of internal docking trials / $short Number of internal docking trials. Only the best model will be written to disk. / $group sampling
```suggestion
    sampling = 2
```

Reply:
The default for ntrials should be 5. And indeed we can reduce the sampling for the test.

Reply:
> And indeed we can reduce the sampling for the test.

keep in mind this might introduce artifcats, it would be better to no do this kind of thing